### DISCUSSION TITLE
Missing Mathematica's stock functions in the suggestion bar while typing

### DISCUSSION THREAD

**Discussion by Hachann** [WLJSTeam/wljs-notebook#279] [Frontend & UI]
Hello,

I noticed `RandomDate[]`, `RandomTime[]`, `PositionSmallest[]`, `PositionLargest[]`, aren't suggested while I'm typing them, even though they are genuine Wolfram Language functions.

Maybe there is a deprecated index of the functions displayed in suggestion bar ?

Thank you !

**Comment by JerryI** [comment]
We always ship the editor with "standard" functions set, so that the server does not have to pump all definitions of the standard library. Last time it was updated on Wolfram Kernel 13. Most probably they were added recently.

**Comment by JerryI** [comment]
Fixed in a new release of wljs-editor
